docs/antrea-l7-network-policy.md

```diff
@@ -11,6 +11,8 @@
   - [TLS](#tls)
     - [More examples](#more-examples-1)
   - [Logs](#logs)
+  - [L7 Visibility](#l7-visibility)
+    - [Usage](#usage-1)
```

Comment:
It's not clear to me why this information is going into this document. Even if the implementations of both features (L7 NP and L7 visibility) are similar and have overlap, they are still 2 distinct features.

Reply:
Should I put it under network-flow-visibility.md file or should it be an independent doc?

Reply:
moved to network-flow-visibility.md